code/game/gamemodes/miniantags/guardian/types/protector.dm

```diff
@@ -66,3 +97,43 @@
 	if(toggle && isliving(mover)) //No crawling under a protector
 		return FALSE
 
+/mob/living/simple_animal/hostile/guardian/protector/Destroy()
+	QDEL_LIST_CONTENTS(connected_shields)
+	return ..()
+
+/obj/effect/guardianshield
+	desc = "A guardian's defensive wall."
+	name = "guardian's shield"
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "shield-grey"
+	can_be_hit = TRUE
+	var/mob/living/simple_animal/hostile/guardian/protector/linked_guardian
+	///If false, the guardian shield is the left shield, otherwise it is the right shield.
+	var/shield_orientation = FALSE
```

Comment:
You could probably make this a little less confusing with a define or something

Reply:
Echoing the above, currently a `TRUE` or `FALSE` doesn't tell you much. A define would work better here, you can perhaps use an existing direction define.

Reply:
Changed to defines